glotaran/utils/io.py

```diff
@@ -252,3 +253,95 @@ def safe_dataframe_replace(
         to_be_replaced_values = [to_be_replaced_values]
     if column_name in df.columns:
         df[column_name].replace(to_be_replaced_values, replace_value, inplace=True)
+
+
+def extract_sas(
+    result: Result | xr.Dataset, species: str | None = None, dataset: str | None = None
+) -> xr.DataArray:
+    """Extract SAS data from a result.
+
+    This is a helper function to easily extract clp guidance spectra.
+
+    Parameters
+    ----------
+    result: Result | xr.Dataset
+        Optimization ``Result`` instance or result dataset.
+    species: str | None
+        Name op the species to extract the SAS for. Defaults to None
```

Comment:
"Name op" -> "Name of"

Reply:
Fixed in [90146e5](https://github.com/glotaran/pyglotaran/pull/1015/commits/90146e5d6c90e55e52795ed357f937e2ac1df19f)